### PR TITLE
Create 2020-05-07-may-meetup.md

### PR DESCRIPTION
**Pull request by LiLietz**
New post for the may 2020 meetup

### CODE REVIEW

File: _posts/2020-05-07-may-meetup.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "May 2020 - Remote again! 💪🏼"
+date:   2020-04-29 19:00:00
```
Comment (JoachimK) [inline _posts/2020-05-07-may-meetup.md]: this should be in the past to make sure it's actually included in the generated html (the preview doesn't currently include it)

File: _posts/2020-05-07-may-meetup.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+In this talk Marin will introduce us to Timelane, his custom instrument that gets integrated directly in Xcode and helps you debug async code with Combine or RxSwift. He will cover the basics like installing, getting accustomed to the UI, and some real-life bug fixing with Timelane.
+
+21:25 - Virtual get together
+
+**I'm in!**
+
+[Meetup RSVP](https://www.meetup.com/CocoaHeads-Hamburg/events/zkldgrybchbkb/)
+
+#### Note on Zoom
+Zoom has a [history of privacy violations and generally shady behavior](https://twitter.com/dhh/status/1244997990382596096). However, it is also currently the most easy-to-setup tool for a group of our size. We'll look for alternatives for future meetups, but until then, we advise you *not to install Zoom on your Mac*. We believe the iOS client to be sufficiently safe due to sandboxing. You can also [use Zoom just from the browser](https://support.zoom.us/hc/en-us/articles/115005666383-Show-a-Join-from-your-browser-Link) ([requires Chrome, audio doesn't work with Firefox + Safari](https://support.zoom.us/hc/en-us/articles/214629443)).
```
Comment (JoachimK) [inline _posts/2020-05-07-may-meetup.md]: let's remove this for the time and add it back in if we do go with Zoom.

File: _posts/2020-04-28-may-meetup.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+As all of you know, we’re stuck at home due to Corona Virus 🦠.
+
+About that, we hope you’re all safe and healthy, as well as you family and friends!
+
+To keep you all distracted and to be able to talk about our favorite nerd topics, we’re not going to give up on our monthly meetup! 💪🏼
+
+As our last months online event worked very well 🎉 we are going to meet remotely again. We are currently evaluating different video call tools to see which one to use and will update here in time.
+
+Also, advices and tipps from you are very welcome! 🙏🏼
```
Comment (JoachimK) [inline _posts/2020-04-28-may-meetup.md]: I think "advice" doesn't have a plural, so it should loose the "s" and it's "tips" in English (but "Tipps" in German... yeah... :D )

File: _posts/2020-04-28-may-meetup.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+This months event is all about Combine!
+
+**Schedule**
+
+19:00 - Virtual "doors" open
+
+Let's gather in Slack!
+
+Click here 👉🏼 https://slack.cocoaheads.hamburg
+to join our workspace 'cocoaheadshamburg.slack.com' to enable an easier communication.
```
Comment (JoachimK) [inline _posts/2020-04-28-may-meetup.md]: Let's make these proper links here, we only don't do it in Meetup as Meetup doesn't support it.

File: _posts/2020-04-28-may-meetup.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+19:30 - Meeting starts 🎉
+
+Everyone joins the video call.
+
+19:45 - 1️⃣ Talk: Learning Combine by refactoring a networking layer - Donny Walls
+In this talk Donny will explain the basics of Combine, and how they can be applied to refactor an existing networking layer into one that uses Combine. Attendees will learn about Combine’s Publishers, Subscribers, operators, the Cancellable protocol and more.
```
Comment (JoachimK) [inline _posts/2020-04-28-may-meetup.md]: You'll need an empty line before this if you want the line-break in the result.

Same for the 2nd talk description

**Review by JoachimK — CHANGES_REQUESTED** [review]

**Review by mariostallone — APPROVED** [review]

**Pull request merged**